high_card.rb

```diff
@@ -1,5 +1,71 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
 
-players = []
+class HighCard
+  attr_accessor :players, :deck, :hands
+  attr_reader :ranks, :suits
+
+  def initialize
+    @ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+    @suits = [ "hearts", "spades", "clubs", "diamonds" ]
+
+    @players = []
+    @deck = []
+    @hands = []
+  end
+
+  def play_game
+    build_deck
+    shuffle_deck
+    get_players
+    deal
+    find_highest_card
+  end
+
+  def new_game
+    self.players = []
```

Comment:
`self` not necessary here since its implied, but continue to use it if it helps you conceptualize better